### PR TITLE
feat: del comment

### PR DESCRIPTION
**Pull request by kjh-busan**
del comment

### CODE REVIEW

File: backend/server.js [hunk 1]
```diff
@@ -1,36 +1,3 @@
-/** CommonJSの場合 */
-// const express = require("express");
-// const connectDB = require("./config/db");
-// const dotenv = require("dotenv");
-/** ES6の場合 */
```
Comment (jugujo) [inline backend/server.js:5]: sdf

**Review by jugujo — COMMENTED** [review]
qwe

File: backend/server.js (hunk 1 above)
Comment (kjh-busan) [reply backend/server.js:5]: OK

Comment (jugujo) [reply backend/server.js:5]: LGTM

**Pull request merged**